lib/core/src/Cardano/Wallet/Transaction.hs

```diff
@@ -401,6 +401,8 @@ data TransactionCtx = TransactionCtx
     -- ^ The assets to mint.
     , txAssetsToBurn :: (TokenMap, Map AssetId (Script KeyHash))
     -- ^ The assets to burn.
+    , txScriptInputs :: Map TxIn (Script KeyHash)
```

Comment:
Perhaps

```
txSimpleScriptInputs :: Map TxIn (Script KeyHash)
```

to distinguish the name from Plutus scripts? 🤔

Reply:
I opted for `txNativeScriptInputs` which is precise naming for this